.drone.yml

```
@@ -11,7 +11,8 @@ workspace:
 
 steps:
   - name: test
-    image: golangci/golangci-lint:v1.38.0-alpine
+    image: golangci/golangci-lint:v2.4.0-alpine
```

If we are making this change; lets also remove alpine which has been removed from other repos.

```suggestion
    image: golangci/golangci-lint:v2.4.0
```